### PR TITLE
ArenaDateHeader and <section> in TabPanel

### PR DESCRIPTION
**Pull request by jeremydthomas**
Pull-Request for `paretOS`

## Description
removed the +1 in getDateRange and getFormattedDay that was causing the days to be one day ahead
in getFor4mattedDay added displayDay param and had it add displayDay. displayDay starts at 0 and goes to 5 when clicking on the days in time travel, which causes the days to move ahead depending on day clicked.  added displayDay to the params in Sprint.js in line 239 ```{getFormattedDay(startDate, displayDay)}```

## Relates to
- #3210

## Reviewers
- @person1 (merge duty)
- @person2
- ...

## Steps to reproduce (if describing bug fix)
- Start project
- Make request to `/endpoint` with `{ "body": "test" }`
- Expect some response

## Screenshots / other info
...
image

the above has day 2 clicked and its showing the 9th

image

### CODE REVIEW

**Comment by mikhael28** [comment]
Hey @jeremydthomas , thank you for sending this in! I will be reviewing this tomorrow, thank you :D

**Comment by jeremydthomas** [comment]
@mikhael28 be gentle, I bruise easily haha jk

File: src/utils/getFormattedDay.ts [hunk 1]
```diff
@@ -1,17 +1,25 @@
 /**
  * This function returns the formatted day.
  * @param startDate date to be formatted
+ * @param displayDay index of the day to display
  * @returns date in the format of "day, month/date"
  * @example getFormattedDay("2020-01-01") returns "Tuesday, 12/17"
  */
-function getFormattedDay(startDate: Date) {
-  const days = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday"];
+function getFormattedDay(startDate: Date, displayDay): string {
```
Comment (mikhael28) [inline src/utils/getFormattedDay.ts:8]: Great job adding the return type!

File: src/arena/Sprint.js [hunk 2]
```diff
@@ -236,7 +236,7 @@ function Sprint(props) {
           <TabPanel value={key} index={1}>
             <>
               <section>
-                <h5>{getFormattedDay(startDate)}</h5>
+                <h5>{getFormattedDay(startDate, displayDay)}</h5>
```
Comment (mikhael28) [inline src/arena/Sprint.js:239]: Great work!

**Review by mikhael28 — APPROVED** [review]
Excellently done! Merging in, no feedback! Just about a perfect solution.

**Pull request merged**